SECURITY-INSIGHTS.yml

```diff
@@ -0,0 +1,55 @@
+header:
+  schema-version: 1.0.0
+  expiration-date: '2024-12-16'
+  project-url: 
+project-lifecycle:
+  bug-fixes-only: false
+  core-maintainers:
```

Comment:
Should the core maintainers point directly to the OWNERS file like here: https://github.com/kubernetes-sigs/kueue/pull/1469/files#diff-000b1100271690c8bb5643a7f806f131a81fdb22cf27af740f5d645e985e9811R17 so we don't have to actively maintain this list here.

Reply:
Yeah, but in this owners file - https://github.com/kubernetes-sigs/karpenter/blob/main/OWNERS, we don't have core maintainers listed, it's mention in this file - https://github.com/kubernetes-sigs/karpenter/blob/main/OWNERS_ALIASES